src/components/list-with-actions/_list-with-actions.scss

```diff
@@ -0,0 +1,63 @@
+@import "../../../../govuk-frontend/govuk/settings/all";
+@import "../../../../govuk-frontend/govuk/tools/all";
+@import "../../../../govuk-frontend/govuk/helpers/all";
+
+.hmrc-list-with-actions {
+  @include govuk-font($size: 19);
+}
+
+// Kept for backward compatibility with add-to-a-list
+.hmrc-list-with-actions--short {
+  @include govuk-media-query($from: desktop) {
+    .hmrc-list-with-actions__name {
+      width: 30%;
+    }
+  }
+}
+
+// Kept for backward compatibility with add-to-a-list
+.hmrc-list-with-actions--long {
+  @include govuk-media-query($from: desktop) {
+    .hmrc-list-with-actions__name {
+      width: 50%;
+    }
+  }
+}
+
+.hmrc-list-with-actions__item {
+  display: flex;
+  justify-content: space-between;
+  padding: govuk-spacing(2) 0;
+  border-bottom: 1px solid $govuk-border-colour;
+}
+
+.hmrc-list-with-actions__name {
+  font-weight: normal;
+}
+
+.hmrc-list-with-actions__actions {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  flex-wrap: wrap;
+
+  @include govuk-media-query($from: desktop) {
+    justify-content: flex-end;
+    flex-direction: row;
+  }
+}
+
+.hmrc-list-with-actions__action {
+  margin-bottom: govuk-spacing(2);
+  text-align: right;
+
+  @include govuk-media-query($from: desktop) {
+    margin-bottom: 0;
+    justify-content: flex-end;
```

Comment:
Looks like this isn't supported in IE11 (https://caniuse.com/mdn-css_properties_justify-content_flex_context_start_end) but I can't see any visual differences when testing. Is this needed?

Reply:
It only makes a difference in the extreme example when the actions wrap onto the next line. However, given we wouldn't encourage this pattern, I suggest we remove for now in the interests of simplicity.